mir_perception/mir_object_recognition/common/src/rgb_object_recognition/__init__.py

```diff
@@ -1,3 +1,5 @@
-from squeezedet_classifier import SqueezeDetClassifier
-from ssdlite_mobilenet import SSDLiteMobilenet
-from utils import util
\ No newline at end of file
+# from .squeezedet_classifier import SqueezeDetClassifier
+# from .ssdlite_mobilenet import SSDLiteMobilenet
+# from .utils import util
+# from image_classifier import ImageClassifier
+# from .yolov7 import yolov7_classifier
```

Comment:
is the last import needed or not? If the others are not used, you can delete them

Reply:
Removed